#todo: figure out what the parameters should be

scan_to_pointcloud:
  ros_parameters:
    min_dist: 0.1
    neighbor_dist: 0.2
    offset: 2.35619449019

control:
  ros_parameters:

magnetometer:
  ros_parameters:

ground_truth:
  ros_parameters:

sim_color_detector:
  ros_parameters:

